extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -201,9 +203,30 @@ export class SqlDatabaseTree {
 
 	private createSearchComponent(): azdata.DivContainer {
 		let resourceSearchBox = this._view.modelBuilder.inputBox().withProps({
+			stopEnterPropagation: true,
 			placeHolder: constants.SEARCH,
 			width: 200
 		}).component();
+		this.focusComponent = resourceSearchBox;
+
+		resourceSearchBox.onTextChanged(value => {
+			const filter: number[] = [];
+			if (this._databaseTableValues && value && value.length > 0) {
+				this._databaseTableValues.forEach((row, index) => {
+					const cell: any = row[1]?.value;
+					const cellText: string = cell?.items[1]?.value?.toLowerCase();
+					const searchText: string = value.toLowerCase();
+					if (cellText?.includes(searchText)) {
+						filter.push(index);
+					}
+				});
+			}
+
+			this._databaseTable.setFilter(
```

Comment:
I'd suggest adding a debounce for this so you aren't sending too many of these if the user is typing really fast